### PR TITLE
Add GrandFinale UI and action handling

### PR DESCRIPTION
**Pull request by sonohoshi**
### Description

SSIA

### Related Links

[그랜드 피날레 클라이언트 개발](https://app.asana.com/0/958521740385861/1203361120589267/f)
[그랜드 피날레 백엔드 개발](https://app.asana.com/0/958521740385861/1203308757840930/f)

### Required Reviewers

@planetarium/9c-dev

### CODE REVIEW

File: nekoyume/Assets/_Scripts/Extensions/AvatarStateExtensions.cs [hunk 1]
```diff
@@ -16,9 +16,11 @@ public static ArenaAvatarState ToArenaAvatarState(this AvatarState avatarState)
         {
             var arenaAvatarState = new ArenaAvatarState(avatarState);
             arenaAvatarState.UpdateCostumes(avatarState.inventory.Costumes
+                .Where(costume => costume.Equipped)
                 .Select(e => e.NonFungibleId)
                 .ToList());
             arenaAvatarState.UpdateEquipment(avatarState.inventory.Equipments
+                .Where(equipment => equipment.Equipped)
```
Comment (boscohyun) [inline nekoyume/Assets/_Scripts/Extensions/AvatarStateExtensions.cs:23]: Could you explain why this `Where` operator added?

File: nekoyume/Assets/_Scripts/BlockChain/ActionRenderHandler.cs [hunk 2]
```diff
@@ -2149,5 +2161,131 @@ private void ResponseClaimRaidReward(ActionBase.ActionEvaluation<ClaimRaidReward
             WorldBossStates.SetReceivingGradeRewards(avatarAddress, false);
             Widget.Find<WorldBossRewardScreen>().Show(new LocalRandom(eval.RandomSeed));
         }
+
+        private void ResponseBattleGrandFinale(ActionBase.ActionEvaluation<BattleGrandFinale> eval)
```
Comment (boscohyun) [inline nekoyume/Assets/_Scripts/BlockChain/ActionRenderHandler.cs:2165]: How about you reusing same logic with `ResponseBattleArena()` method?

File: nekoyume/Assets/_Scripts/Extensions/AvatarStateExtensions.cs (hunk 1 above)
Comment (sonohoshi) [reply nekoyume/Assets/_Scripts/Extensions/AvatarStateExtensions.cs:23]: This is to fix a bug that indicates that all items in the inventory are equipped.

File: nekoyume/Assets/_Scripts/BlockChain/ActionRenderHandler.cs (hunk 2 above)
Comment (sonohoshi) [reply nekoyume/Assets/_Scripts/BlockChain/ActionRenderHandler.cs:2165]: I think that's a good suggestion. However, I think it is good to manage it even if it overlaps right now.

**Review by boscohyun — APPROVED** [review]

**Pull request merged**